CHANGELOG.md

```diff
@@ -9,7 +9,11 @@ Every entry has a category for which we use the following visual abbreviations:
 - ⚠️ change
 - 🐞 bugfix
 
-<!-- ## Unreleased -->
+## Unreleased
+
+- ⚠️ On Linux systems, VAST no longer looks for the `share/vast` directory
+  relative to the binary location. On macOS and FreeBSD, this behavor still
+  exists. [#1036](https://github.com/tenzir/vast/pull/1036)
```

Comment:
```suggestion
- ⚠️ On Linux, VAST no longer looks for the `share/vast` directory
  relative to the location of the `vast` binary. On macOS and FreeBSD, this behavior still
  exists. [#1036](https://github.com/tenzir/vast/pull/1036)
```

I'm mostly missing a reason for why that change occurred in the changelog.

Reply:
I improved the wording. Please take another look.